nexus/mgs-updates/src/host_phase1_updater.rs

```diff
@@ -133,6 +133,8 @@ use crate::SpComponentUpdateHelperImpl;
 use crate::common_sp_update::PostUpdateError;
 use crate::common_sp_update::PrecheckError;
 use crate::common_sp_update::PrecheckStatus;
+use crate::sp_updater::WAIT_FOR_SP_STATE_TIMEOUT;
```

Comment:
I'm not sure if we should use the same timeout/interval as when updating the SP. Thoughts?